.storybook/test-runner.ts

```diff
@@ -71,19 +66,20 @@ module.exports = {
         jest.retryTimes(RETRY_TIMES, { logErrorsBeforeRetry: true })
         jest.setTimeout(JEST_TIMEOUT_MS)
     },
-    async postRender(page, context) {
+    async postVisit(page, context) {
```

Comment:
@daibhin This is what we talked about referring to `postRender` vs. interaction tests. My thinking was wrong, because apparently `postRender` is in fact ran immediately after render and doesn't wait for anything else to play out. But the new `postVisit` does!